radio/src/mixer.cpp

```diff
@@ -947,7 +947,7 @@ void evalMixes(uint8_t tick10ms)
         mixerCurrentFlightMode = p;
         evalFlightModeMixes(p==fm ? e_perout_mode_normal : e_perout_mode_inactive_flight_mode, p==fm ? tick10ms : 0);
         for (uint8_t i=0; i<MAX_OUTPUT_CHANNELS; i++)
-          sum_chans512[i] += (chans[i] >> (17 - __builtin_clz(abs(chans[i])))) * fp_act[p];
+          sum_chans512[i] += (chans[i] >> (__builtin_clz(abs(chans[i])) >= 17 ? 0 : 17 - __builtin_clz(abs(chans[i])))) * fp_act[p];
```

Comment:
is it possible you create a macro to hide this? it has become difficult to read

Reply:
may I ask you to add a new non regression test for this bug?

Reply:
Sure